BSB_LAN/include/get_cmdtbl_values.h

```diff
@@ -53,6 +53,44 @@ uint16_t get_cmdtbl_line(int i) {
   return l;
 }
 
+float get_next_prognr(float currentProgNR, int startFromTableLine){
+  if(startFromTableLine < 0) {
+    float intpart;
+    modf(currentProgNR, &intpart);
+    return intpart + 1;
+  }
+  int cmdtblsize = sizeof(cmdtbl1)/sizeof(cmdtbl1[0]) + sizeof(cmdtbl2)/sizeof(cmdtbl2[0]) + sizeof(cmdtbl3)/sizeof(cmdtbl3[0]);
```

Comment:
Just a question: The merging of cmdtbl1/2/3 into one cmdtbl is done in a different step, right?